prow/cmd/deck/main.go

```
@@ -574,7 +590,9 @@ func renderSpyglass(sg *spyglass.Spyglass, ca *config.Agent, src string, staticF
 		Source:      src,
 		ViewerCache: viewerCache,
 	}
-	t, err := template.ParseFiles(path.Join(staticFilesLocation, "template/spyglass-template.html"))
+	t := template.New("spyglass.html")
+	prepareBaseTemplate(ca, t)
```

we're ignoring the error here